### ISSUE TITLE
Similar project in Rust?

### ISSUE THREAD

**Issue by cdbattags** [teragonaudio/MrsWatson#282]
Anyone know if there is a similar project to MrsWatson but using Rust?

Thanks!

**Comment by harritaylor** [comment]
No, but I'd be interested in working on one. I'm aware of: https://github.com/rust-dsp/rust-vst/ which would be a good place to start. (Sorry if this is unhelpful!)
